examples/plot_0_sin.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+"""
+Introductory example - Plotting sin
+===================================
+
+This is a general example demonstrating a Matplotlib plot output, embedded
+rST, the use of math notation and cross-linking to other examples. It would be
+useful to compare the :download:`source Python file <plot_0_sin.py>` with the
+output below.
+
+Example files should start with a header docstring. Anything before the
+docstring is ignored by Sphinx-Gallery and will not appear in the rendered
+output, nor will it be executed. This docstring requires an rST header, used
+at the title of the example and to correctly build reference links.
+
+Once you close the docstring you will be writing Python code. This code gets
+executed by Sphinx-Gallery and any output, including plots will be captured.
+You can also alternate between code blocks and text blocks that provide
+explanations about the code.
+
+To include displayed math notation, use the directive ``.. math::`` and to
+include inline math notation use ``:math:``. For example, we wish to plot the
+following function:
+
+.. math::
+
+    x \\rightarrow \\sin(x)
+
+Here the function :math:`\\sin` is evaluated at each point the variable
+:math:`x` is defined. When writting latex in a Python string, ensure that you
+escape the backslashes or use a raw docstring. You do not need to do
```

Comment:
```suggestion
escape the backslashes or use a :py:std:label:`raw docstring <strings>`.
You do not need to do
```

Reply:
```
:py:std:label:`raw docstring <strings>`
```
this doesn't render as anything. What is it meant to be..?

I can simply add an example i.e., `r'<string>'`

Reply:
Now I know why I can't add ``r"""<docstring>"""`` - SG no longer picks up the docstring if I put that in the docstring. I tried a hacky substitution (e.g. `.. |docstring| replace:: """`) but it didn't work.
Suggestions?

Reply:
Probably needs to be explicit about where the intersphinx ref is (`python`) and/or remove the `:py:` label (it's just under `:std:label:` in intersphinx), so:
```
:std:label:`raw docstring <python:strings>`
```
Verified that it's likely to work by doing:
```
python -m sphinx.ext.intersphinx https://docs.python.org/3/objects.inv > ~/Desktop/python.txt
```
Then looking to see that it's under the `std:label` category.

I would make this change locally then `make html` to see if it works.

Reply:
```
:std:label:`raw docstring <python:strings>`
```
didn't work. Am I doing something wrong?

Reply:
I don't need to update `intersphinx_mapping` in the `conf.py` file right?

Reply:
`:python:std:label:`raw docstring <python:strings>` also didn't work.

Reply:
Shouldn't need to as there is already an entry for `python` there. I'll try a few things locally in a little while

Reply:
@larsoner 
```
:ref:`raw docstring <python:strings>`
```
works and links to [this page](https://docs.python.org/3/reference/lexical_analysis.html#strings), which I think is correct (unfamiliar with intersphinx and versions)

Reply:
That looks correct